src/us/kbase/workspace/database/provenance/EventDate.java

```diff
@@ -62,34 +65,7 @@ public boolean equals(Object obj) {
 	}
 
 	/**
-	 * Builds an {@link EventDate}.
-	 *
-	 * @param date  the date when the event occurred,
-	 *              in the format yyyy-MM-dd, yyyy-MM, or yyyy.
-	 * @param event the event that occurred on that date, as a string.
-	 *              See the {@link Event} class for valid string values.
-	 * @return the new {@link EventDate}.
-	 */
-	public static EventDate build(final String date, final String event) {
-		final Event eventObject = Event.getEvent(Util.checkString(event, "event"));
-		return build(date, eventObject);
-	}
-
-	/**
-	 * Builds an {@link EventDate}.
-	 *
-	 * @param date  the date when the event occurred,
-	 *              in the format yyyy-MM-dd, yyyy-MM, or yyyy.
-	 * @param event the event that occurred on that date, as an Event
-	 * @return the new {@link EventDate}.
-	 */
-	public static EventDate build(final String date, final Event event) {
-		Objects.requireNonNull(event, "event cannot be null");
-		final String protoDate = checkDate(Util.checkString(date, "date"));
-		return new EventDate(protoDate, event);
-	}
-
-	/** Ensures that a string can be parsed as a valid date.throws
+	 * Ensures that a string can be parsed as a valid date.
```

Comment:
Moved the `parseDate` function above the `build` functions so the object constructors are at the bottom of the file.